src/Hilke.KineticConvolution/DoubleNumber.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+
+namespace Hilke.KineticConvolution
+{
+    public sealed class DoubleNumber : IAlgebraicNumber<DoubleNumber>, IEquatable<DoubleNumber>
+    {
+        private DoubleNumber(double value) => Value = value;
+
+        public double Value { get; }
+
+        public DoubleNumber Add(DoubleNumber operand) => new DoubleNumber(operand.Value + Value);
+
+        public DoubleNumber Subtract(DoubleNumber operand) => new DoubleNumber(Value - operand.Value);
+
+        public DoubleNumber Multiply(DoubleNumber operand) => new DoubleNumber(operand.Value * Value);
+
+        public DoubleNumber Divide(DoubleNumber operand) => new DoubleNumber(Value / operand.Value);
```

Comment:
Uniformiser l'ordre des variables dans les opérations (si le résultat est bien équivalent) :

```suggestion
        public DoubleNumber Add(DoubleNumber operand) => new DoubleNumber(Value + operand.Value);

        public DoubleNumber Subtract(DoubleNumber operand) => new DoubleNumber(Value - operand.Value);

        public DoubleNumber Multiply(DoubleNumber operand) => new DoubleNumber(Value * operand.Value);

        public DoubleNumber Divide(DoubleNumber operand) => new DoubleNumber(Value / operand.Value);
```

Reply:
done